extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -60,6 +61,7 @@ amp-story-shopping-tag {
   justify-content: center !important;
   background-color: rgba(0, 0, 0, 0.6) !important;
   border-radius: 100% !important;
+  flex-shrink: 0 !important;
```

Comment:
Would setting `flex-shrink` to 0 cause other elements within the shopping tag (e.g., any text) to shrink in order to give space to the dot and image? I'm not familiar with `flex-shrink`, so I'm just basing my question on what I read [here](https://cssreference.io/property/flex-shrink/).

Reply:
This is correct and intended since the text area can grow and shrink based on content.